src/ngAnimate/animateQueue.js

```diff
@@ -491,6 +491,12 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
       var animateChildren;
 
       while (parent && parent.length) {
+        if (!rootElementDetected) {
+          // angular doesn't want to attempt to animate elements outside of the application
+          // therefore we need to ensure that the rootElement is an ancestor of the current element
+          rootElementDetected = isMatchingElement(parent, $rootElement);
+        }
```

Comment:
y>3x²
On Apr 14, 2015 2:32 PM, "Matias Niemelä" <notifications@github.com> wrote:

> In src/ngAnimate/animateQueue.js
> <https://github.com/angular/angular.js/pull/11575#discussion_r28303219>:
>
> > @@ -491,6 +491,12 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
> >        var animateChildren;
> >
> >        while (parent && parent.length) {
> > +        if (!rootElementDetected) {
> > +          // angular doesn't want to attempt to animate elements outside of the application
> > +          // therefore we need to ensure that the rootElement is an ancestor of the current element
> > +          rootElementDetected = isMatchingElement(parent, $rootElement);
> > +        }
>
> The nodeType if statement still breaks the loop, but the rootElement is
> detected beforehand now so that sets rootElementDetected to true.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/angular.js/pull/11575/files#r28303219>.
>